File: libraries/analytics_debugger_noop/src/main/java/com/tokopedia/analyticsdebugger/debugger/TopAdsLoggerInterface.java
package com.tokopedia.analyticsdebugger.debugger;

public interface TopAdsLoggerInterface {

    boolean isNotificationEnabled();

    void save(String url, String eventType, String sourceName, String productId, String productName,
              String imageUrl, String componentName);

    void openActivity();

    void enableNotification(boolean status);

}

File: libraries/analytics_debugger/src/main/java/com/tokopedia/analyticsdebugger/debugger/TopAdsLoggerInterface.kt
package com.tokopedia.analyticsdebugger.debugger

interface TopAdsLoggerInterface {

    val isNotificationEnabled: Boolean

    fun save(url: String,
             eventType: String,
             sourceName: String,
             productId: String,
             productName: String,
             imageUrl: String,
             componentName: String
    )

    fun openActivity()

    fun enableNotification(status: Boolean)
}
